org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/AbstractPlugin.java

```diff
@@ -22,6 +22,10 @@
 import org.eclipse.core.runtime.Plugin;
 import org.osgi.framework.BundleContext;
 
+/**
+ * @deprecated since 2.7 use {@link Plugin} directly
+ */
+@Deprecated
 public class AbstractPlugin extends Plugin {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)